homeassistant/components/sensor/dte_energy_bridge.py

```diff
@@ -0,0 +1,82 @@
+"""Support for monitoring energy usage using the DTE energy bridge."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+ICON = 'mdi:flash'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the DTE energy bridge sensor."""
+    ip_address = config.get('ip')
+    if not ip_address:
```

Comment:
Please use the [validate_config helper function](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/__init__.py#L7) for this.